tests/braille-specs/he-common-characters.yaml

```diff
@@ -0,0 +1,21 @@
+
+# Yaml Test For hebrew-common-characters
+
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+display: both
+table:
+  language: he-common-characters
+  __assert-match: he-common-characters.uti
+flags: { testmode: forward }
+tests:
+  - ["בְּרֵא֯שִׁית", "⠃⠄⠗⠌⠁֯⠩⠊⠚⠹"]
+  - ["בָּרָא", "⠃⠣⠗⠣⠁"]
+  - ["אֱלֹהִים ", "⠁⠢⠇⠕⠓⠔⠍ "]
+  - ["הָאָרֶץ.", "⠓⠣⠁⠣⠗⠑⠋⠮"]
+  - ["הָיְתָה תֹה. ", "⠓⠣⠚⠄⠹⠣⠓ ⠹⠕⠓⠲ "]
+  - ["“וְהָאָרֶץ “‬", "“⠺⠄⠓⠣⠁⠣⠗⠑⠮ “‬"]
+  - ["“הַמָּיִם.”", "“⠓⠉⠐⠍⠣⠚⠊⠍⠲”"]
```

Comment:
@yeatersink Most of these tests fail. Some even contain characters that are not Unicode braille, so there's definitely something wrong with the expected braille. Could you please check them? I needed I can give you the actual braille that Liblouis returns.

Reply:
Hi @yeatersink 

> The YAML file i for the He-common characters.yml had several unnecessary quotes and spaces that I believe were causing the errors. I make the necessary changes, I hope this fixes the problem?

This fixed some of the failures, but we were still left with three:

1. Input:    'הָאָרֶץ.'
   Expected: '⠓⠣⠁⠣⠗⠑⠮⠲'
   Received: '⠓⠣⠁⠣⠗⠑⠮⠄⡳⠭⠴⠴⠆⠑⠄'

2. Input:    'הָיְתָהתֹה'
   Expected: '⠓⠣⠚⠄⠹⠣⠓⠹⠕⠓⠲'
   Received: '⠓⠣⠚⠄⠹⠣⠓⠹⠕⠓'

3. Input:    'הַמָּיִם.'
   Expected: '⠓⠉⠐⠍⠣⠚⠊⠍⠲'
   Received: '⠓⠉⠐⠍⠣⠚⠊⠍⠄⡳⠭⠴⠴⠆⠑⠄'

In your last commit you changed the 'הָיְתָהתֹה' test to 'הָיְתָה', and now that one passed. But 1. still gives the same result, and 3. has changed but still fails:

- Input:    'הַמָּיִם.'
  Expected: '⠓⠉⠐⠍⠣⠔⠍⠲'
  Received: '⠓⠉⠐⠍⠣⠔⠍⠄⡳⠭⠴⠴⠆⠑⠄'

The problem with these two tests is the period which is not defined in he-common.uti, so it produces the escape sequence '⠄⡳⠭⠴⠴⠆⠑⠄' instead of '⠲'. Some rules in he-common.uti contain a period, but do not match. For example, the last test would presumably pass if the this rule would not have a `nofor` prefix:

    nofor always ם. 134-256  # final Mem with  period

I have fixed the test by, instead of testing he-common.uti, test a table that includes he-common.uti and defines the period as follows: 

    punctuation . 256
    include he-common.uti

Everything good now in he-common-characters.yaml...

Reply:
I don't see a commit? Did you forget to push?

Reply:
I've got it all locally.